docs/rest_api.md

```diff
@@ -175,6 +172,102 @@ Additionally, the `show_networks` api call may return the channel identifier
 Where documentation specifies that the network driver should choose a
 default channel, the VLAN drivers choose `vlan/native`.
 
+#### list_networks
+
+`GET /networks`
+
+List all networks.
+
+Returns a JSON dictionary of dictionaries, where the exterior dictionary is indexed by
+the network name and the value of each key is another dictionary with keys corresponding
+to that network's id and projects
+
+The response must contain the following fields:
+
+* "network", the name of a network
+* "network_id", the id of the network
+* "projects", a list of projects with access to the network or 'None' if network is public
+
+Example Response:
+	{
+		"netA": {
+			"network_id": "101",
+			"projects": ["qproj-01", qproj-02"]
+			},
+		"netB": {
+			"network_id": "102",
+			"projects": None}
+	}
+
+Authorization requirements:
+
+Administrative access is required
+
+#### list_network_attachments
+
+`GET /network/<network>/attachments`
+
+List all nodes that are attached to network <network>.
+
+If optional argument 'project' is supplied will only list attached nodes
+belonging to the specified project.
```

Comment:
*paraphrase:*
If optional argument 'project' is supplied, only attached nodes belonging to the specified project will be listed.